src/test/unit/algorithms/TemporalMemoryTest.cpp

```diff
@@ -1357,16 +1357,16 @@ TEST(TemporalMemoryTest, testNumberOfColumns) {
   TemporalMemory tm;
   tm.initialize(vector<UInt>{64, 64}, 32);
 
-  size_t numOfColumns = tm.numberOfColumns();
-  ASSERT_EQ(numOfColumns, 64ull * 64ull);
+  int numOfColumns = tm.numberOfColumns();
```

Comment:
nit, do not change here, but in tests let's use `auto` as much as possible. So when we change the algorithm, the tests would keep working.